src/includes/getting-started-config/capacitor.mdx

```diff
@@ -0,0 +1,19 @@
+You should `init` the SDK as soon as possible during your app's creation stage.
+
+```javascript
+import * as Sentry from "@sentry/capacitor";
+import { Integrations } from "@sentry/tracing";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // To set your release version
+  release: "my-project-name@" + process.env.npm_package_version,
+  integrations: [new Integrations.BrowserTracing()],
```

Comment:
It supports transactions completely on Android with BrowserTracing?

@jennmueng this approach didn't work out on RN, right?

Reply:
@bruno-garcia It'll only have Fetch/XHR and will log [a warning](https://github.com/getsentry/sentry-javascript/blob/d8ecb2b927e23c0583456bef0b0c04d01dfe5eba/packages/tracing/src/browser/router.ts#L14-L17) when it trues to instrument routing

Reply:
Unless capacitor has browser's `location` global?

Reply:
@bruno-garcia good catch, I meant to take this out!